### PR TITLE
Add import and export CSV flow for Members

### PR DESCRIPTION
**Pull request by Guccio163**
### Details

This PR adds the import and export logic to the `Members` sections of the workspace. This PR takes into account all raised and fixed bugs for `Category` section's import/export functions. 

Fyi: This PR is created in parallel with PR for `Tags` import/export which operates on similar files - depending on which gets merged earlier there will be few minor conflicts to take care of.

### Fixed Issues

$ https://github.com/Expensify/App/issues/47869
PROPOSAL:

### Tests

Import:
1. Go to any Workspace -> Categories
2. Press three dots (in header)
3. Press Import spreadsheet
4. Press Choose file
5. Select xls, xlsx, csv or txt file
6. Select columns to import
7. Press import
8. Selected column of chosen file should be imported

Export:
1. Navigate into workspace.
2. Enter Members section.
3. Add some members if you don'e have any.
4. Click the three-dot icon in top-right corner (Import Spreadsheet and Download CSV options should appear)
5. Select Download CSV.
6. "Members-[time] [date]" file should be downloaded with all members in .csv format.

- [x] Verify that no errors appear in the JS console

### Offline tests

N/A

### QA Steps

N/A

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/72494aad-d59e-4b60-9aaf-3da7e1d99cb6

https://github.com/user-attachments/assets/696ed4b5-d0e8-491d-be2f-32ee41d36c9b

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/de1be13f-794e-4323-ae93-ac122834d69d

https://github.com/user-attachments/assets/ccbfdd14-36ed-4dde-984c-b0d9e4bbd937

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/8a269a20-edc2-417c-932b-de332791fd02

https://github.com/user-attachments/assets/d35568ec-68ba-4feb-969e-e5979e9ecffb

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/1bac4b4b-6df6-4cac-b265-cfed33441db0

https://github.com/user-attachments/assets/8f9c9e1f-1aaf-4da8-9b5f-f894c3ca62e8

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/38117275-357a-43ea-9753-0b5703276f0b

https://github.com/user-attachments/assets/e1fce270-02cf-41a6-bb97-0e97b62ddc4e

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/56d2c3c1-ccf8-45ff-8738-f3d438746970

https://github.com/user-attachments/assets/cd4f1ffe-9e93-4243-8017-ad1e12792481

</details>

### CODE REVIEW

File: src/languages/es.ts [hunk 1]
```diff
@@ -676,6 +676,7 @@ export default {
         importFailedTitle: 'Fallo en la importación',
         importFailedDescription: 'Por favor, asegúrate de que todos los campos estén llenos correctamente e inténtalo de nuevo. Si el problema persiste, por favor contacta a Concierge.',
         importCategoriesSuccessfullDescription: (categories: number) => (categories > 1 ? `Se han agregado ${categories} categorías.` : 'Se ha agregado 1 categoría.'),
+        importMembersSuccessfullDescription: (members: number) => (members > 1 ? `Se han agregado ${members} miembros.` : 'Se ha agregado 1 miembro.'),
```
Comment (blazejkustra) [inline src/languages/es.ts:679]: These translations are already approved?

File: src/libs/actions/Policy/Member.ts [hunk 2]
```diff
@@ -834,6 +885,20 @@ function declineJoinRequest(reportID: string, reportAction: OnyxEntry<ReportActi
 
     API.write(WRITE_COMMANDS.DECLINE_JOIN_REQUEST, parameters, {optimisticData, failureData, successData});
 }
+function downloadMembersCSV(policyID: string) {
```
Comment (blazejkustra) [inline src/libs/actions/Policy/Member.ts:888]: Nitpick:
```suggestion

function downloadMembersCSV(policyID: string) {
```

File: src/libs/actions/Policy/Member.ts [hunk 3]
```diff
@@ -639,6 +674,22 @@ function addMembersToWorkspace(invitedEmailsToAccountIDs: InvitedEmailsToAccount
...
+    email: string;
+    role: string;
+};
+
+function importPolicyMembers(policyID: string, members: PolicyMember[]) {
+    const onyxData = updateImportSpreadsheetData(members.length);
+
+    const parameters = {
+        policyID,
+        employees: JSON.stringify([...members.map((member) => ({email: member.email, role: member.role}))]),
```
Comment (blazejkustra) [inline src/libs/actions/Policy/Member.ts:687]: NAB:
```suggestion
        employees: JSON.stringify(members.map((member) => ({email: member.email, role: member.role}))),
```

File: src/libs/actions/Policy/Member.ts [hunk 4]
```diff
@@ -166,6 +171,36 @@ function buildAnnounceRoomMembersOnyxData(policyID: string, accountIDs: number[]
     });
     return announceRoomMembers;
 }
+/**
+ * Updates the import spreadsheet data according to the result of the import
+ */
+function updateImportSpreadsheetData(membersLength: number) {
```
Comment (blazejkustra) [inline src/libs/actions/Policy/Member.ts:177]: NAB:
```suggestion
function updateImportSpreadsheetData(membersLength: number): OnyxData {
```

File: src/pages/workspace/members/ImportedMembersPage.tsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+    const spreadsheetColumns = spreadsheet?.data;
+    if (!spreadsheetColumns) {
+        return;
+    }
+
+    const closeImportPageAndModal = () => {
+        setIsImporting(false);
+        closeImportPage();
+        Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES.getRoute(policyID));
```
Comment (blazejkustra) [inline src/pages/workspace/members/ImportedMembersPage.tsx:129]: I think `WORKSPACE_CATEGORIES` is a mistake, right?

**Review by blazejkustra — APPROVED** [review]
Overall looks good! 👍

File: src/pages/workspace/members/ImportedMembersPage.tsx (hunk 5 above)
Comment (Guccio163) [reply src/pages/workspace/members/ImportedMembersPage.tsx:129]: Yes of course, I missed it when copy pasting haha!

File: src/languages/es.ts (hunk 1 above)
Comment (Guccio163) [reply src/languages/es.ts:679]: I'm not quite sure and I can't find a message requesting a translation approval, I'll check this!

Comment (Guccio163) [reply src/languages/es.ts:679]: I believe that Filip checked the translations before previous PR that handled `Categories` import, these are generally the same except 'Members' and 'Tags' names

**Comment by Guccio163** [comment]
@Expensify/design PR ready to check out

**Comment by blazejkustra** [comment]
@rushatgabhane in case of conflicts/change requests please ping me, I'm going to take over from Wiktor as he will be OOO in the next days 😄

**Comment by blazejkustra** [comment]
@rushatgabhane When do you think you'll be able to finish the review on this one?

**Comment by rushatgabhane** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Review by rushatgabhane — APPROVED** [review]
LGTM

**Review by rushatgabhane — APPROVED** [review]

**Comment by blazejkustra** [comment]
@madmax330 Merged the newest main for you 😄

File: src/pages/workspace/members/ImportedMembersPage.tsx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+function findDuplicate(array: string[]): string | null {
+    const frequencyCounter: Record<string, number> = {};
+
+    for (const item of array) {
+        if (item !== CONST.CSV_IMPORT_COLUMNS.IGNORE) {
+            if (frequencyCounter[item]) {
+                return item;
+            }
+            frequencyCounter[item] = (frequencyCounter[item] || 0) + 1;
```
Comment (madmax330) [inline src/pages/workspace/members/ImportedMembersPage.tsx:43]:
```suggestion
            frequencyCounter[item] = 1;
```

Comment (madmax330) [reply src/pages/workspace/members/ImportedMembersPage.tsx:43]: This should still work right since we just check above if it exists?

File: src/pages/workspace/members/ImportedMembersPage.tsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+        {text: translate('common.role'), value: CONST.CSV_IMPORT_COLUMNS.ROLE},
+    ];
+
+    const requiredColumns = columnRoles.filter((role) => role.isRequired).map((role) => role);
+
+    const validate = useCallback(() => {
+        const columns = Object.values(spreadsheet?.columns ?? {});
+        let errors: Record<string, string | null> = {};
+
+        if (!requiredColumns.every((requiredColumn) => columns.includes(requiredColumn.value))) {
```
Comment (madmax330) [inline src/pages/workspace/members/ImportedMembersPage.tsx:71]: Can we add a comment here? This code is really dense and hard to follow

File: src/pages/workspace/members/ImportedMembersPage.tsx (hunk 6 above)
Comment (Guccio163) [reply src/pages/workspace/members/ImportedMembersPage.tsx:43]: Yes I think it should work no matter which truth value we assign to the `frequencyCounter[item]` 👍

File: src/pages/workspace/members/ImportedMembersPage.tsx (hunk 7 above)
Comment (Guccio163) [reply src/pages/workspace/members/ImportedMembersPage.tsx:73]: Yes, no problem; what do you think about this comment?

File: src/libs/API/parameters/ExportMembersSpreadsheetParams.ts [hunk 8]
```diff
@@ -0,0 +1,6 @@
+type ExportMemberssSpreadsheetParams = {
```
Comment (rlinoz) [inline src/libs/API/parameters/ExportMembersSpreadsheetParams.ts:1]:
```suggestion
type ExportMembersSpreadsheetParams = {
```

File: src/libs/API/parameters/ExportMembersSpreadsheetParams.ts [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+    /** ID of the policy */
+    policyID: string;
+};
+
+export default ExportMemberssSpreadsheetParams;
```
Comment (rlinoz) [inline src/libs/API/parameters/ExportMembersSpreadsheetParams.ts:6]:
```suggestion
export default ExportMembersSpreadsheetParams;
```

File: src/pages/workspace/members/ImportedMembersPage.tsx [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+        if (item !== CONST.CSV_IMPORT_COLUMNS.IGNORE) {
+            if (frequencyCounter[item]) {
+                return item;
+            }
+            frequencyCounter[item] = 1;
+        }
+    }
+
+    return null;
+}
```
Comment (rlinoz) [inline src/pages/workspace/members/ImportedMembersPage.tsx:48]: We moved all this to [importSpreadsheetUtils](https://github.com/Expensify/App/blob/main/src/libs/importSpreadsheetUtils.ts), let's use it from there.

**Review by rlinoz — CHANGES_REQUESTED** [review]
This is looking good! Just a few comments.

**Comment by Guccio163** [comment]
Thanks for review @rlinoz, I also merged newest main when applying changes

File: src/languages/en.ts [hunk 11]
```diff
@@ -3260,6 +3261,9 @@ export default {
             addedWithPrimary: 'Some members were added with their primary logins.',
             invitedBySecondaryLogin: ({secondaryLogin}) => `Added by secondary login ${secondaryLogin}.`,
             membersListTitle: 'Directory of all workspace members.',
+            importMembers: 'Import members',
+            importedMembersMessage: (columnCounts: number) =>
+                `We found *${columnCounts} columns* in your spreadsheet. Select *Email* next to the column that contains emails. You can also select *Role* next to the column that sets users roles.`,
```
Comment (rlinoz) [inline src/languages/en.ts:3266]: @Guccio163 One last thing from me, was this text agreed somewhere? I don't remember where we discussed texts.

If so, I think everything that is between `*` should actually yield a bold text
image

Comment (Guccio163) [reply src/languages/en.ts:3266]: I can't be sure about that because I took that task from @filip-solecki with import functionality already made (and hence translations). To be fair I cannot find any conversation touching this topic so can you please assign someone from translation team to check these? I'll also check how this text should look like from doc, hopefully `*` will be explained.

Comment (rlinoz) [reply src/languages/en.ts:3266]: Nice, coming from [this thread](https://expensify.slack.com/archives/C06ML6X0W9L/p1726603502772629?thread_ts=1726494652.467189&cid=C06ML6X0W9L), let's use the same message we use for categories.

image

Comment (Guccio163) [reply src/languages/en.ts:3270]: @rlinoz Changes applied, it turns out that it was fixed in the whole `ImportSpreadsheetColumns' component in `Tags` PR mentioned above, which standardises header in all import sections 👌

Comment (rlinoz) [reply src/languages/en.ts:3270]: @Guccio163 I guess we can remove this line then right? I don't see it being used anywhere else.

Comment (Guccio163) [reply src/languages/en.ts:3270]: Oh yes, I knew I forgot something!

Comment (Guccio163) [reply src/languages/en.ts]: @rlinoz fixed 👍

**Review by rlinoz — APPROVED** [review]
Nice work!

**Comment by Guccio163** [comment]
@madmax330 just LYK I think it's ready for merge

**Comment by Guccio163** [comment]
@madmax330 bumping, ready for merge

**Review by madmax330 — APPROVED** [review]

**Comment by madmax330** [comment]
All tests were passing

File: src/pages/workspace/WorkspaceMembersPage.tsx [hunk 12]
```diff
@@ -531,7 +535,7 @@ function WorkspaceMembersPage({personalDetails, route, policy, currentUserPerson
             return null;
         }
         return (
-            <View style={styles.w100}>
+            <View style={[styles.flexRow, styles.gap2, shouldUseNarrowLayout && styles.mb3]}>
```
Comment (aldo-expensify) [inline src/pages/workspace/WorkspaceMembersPage.tsx:538]: This seems to have caused this: https://github.com/Expensify/App/issues/49749, why did we change the style here? @Guccio163

Comment (aldo-expensify) [reply src/pages/workspace/WorkspaceMembersPage.tsx:538]: Was it because we needed to acomodate two buttons?

image

Comment (aldo-expensify) [reply src/pages/workspace/WorkspaceMembersPage.tsx:538]: oh, I see, without this change in desktop the 3-vertical-dots are too far to the right:

image

File: src/pages/workspace/members/ImportMembersPage.tsx [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type ImportMembersPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.MEMBERS_IMPORT>;
+
+function ImportMembersPage({route}: ImportMembersPageProps) {
+    const policyID = route.params.policyID;
+
+    return (
+        <ImportSpreedsheet
```
Comment (Pujan92) [inline src/pages/workspace/members/ImportMembersPage.tsx:14]: We need to wrap it with `AccessOrNotFoundWrapper` to avoid issue https://github.com/Expensify/App/issues/51129

File: src/libs/actions/Policy/Member.ts [hunk 14]
```diff
@@ -167,6 +172,36 @@ function buildAnnounceRoomMembersOnyxData(policyID: string, accountIDs: number[]
...
+ */
+function updateImportSpreadsheetData(membersLength: number): OnyxData {
+    const onyxData: OnyxData = {
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.IMPORTED_SPREADSHEET,
+                value: {
+                    shouldFinalModalBeOpened: true,
+                    importFinalModal: {title: translateLocal('spreadsheet.importSuccessfullTitle'), prompt: translateLocal('spreadsheet.importMembersSuccessfullDescription', membersLength)},
```
Comment (alitoshmatov) [inline src/libs/actions/Policy/Member.ts:186]: Not sure if it is considered a bug, but we improved the messages to show exactly how many members were added and how many updated: https://github.com/Expensify/App/issues/56941#issuecomment-2663426767
